person.rb

```diff
@@ -7,12 +9,17 @@ def initialize(id, age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super
```

Comment:
- [ ] There was also an issue running your app as a result of this `super`. Kindly resolve it.